action.yml

```diff
@@ -1,10 +1,14 @@
 name: 'Install HLint to be used with hlint-run'
 description: 'Cached download of hlint binary release to be used with hlint-run'
 inputs:
+  token:
+    description: The token to use when communicating with GitHub's API.
+    required: false
+    default: ${{ github.token }}
```

Comment:
This defaults to a read-only token, which is enough to avoid being rate limited by GitHub's API. See these pull requests for how I discovered and worked through this with cabal-gild-setup-action:

- https://github.com/tfausak/cabal-gild-setup-action/pull/2
- https://github.com/tfausak/cabal-gild-setup-action/pull/3

Reply:
Thank you for this!

I was planning to review the token permissions at some point to reduce them to minimal.